### PR TITLE
Feat/user invite

### PR DESCRIPTION
**Pull request by speed2exe**
- non admin user can invite other user
- an email will be sent to the invited user, containing a login link
- added login for github and discord
- added ability for users to navigate to app from web
- added pgadmin, minio and portainer quick links in the admin page

### CODE REVIEW

File: admin_frontend/src/web_api.rs [hunk 1]
```diff
@@ -21,23 +25,77 @@ pub fn router() -> Router<AppState> {
...
+) -> Result<WebApiResponse<WebApiOpenAppResponse>, WebApiError<'static>> {
+  let access_token_resp = state
+    .gotrue_client
+    .token(&Grant::RefreshToken(RefreshTokenGrant {
+      refresh_token: session.refresh_token.to_owned(),
+    }))
+    .await?;
+
+  let app_sign_in_url = format!(
+      "appflowy-flutter://login-callback#access_token={}&expires_at={}&expires_in={}&refresh_token={}&token_type={}",
```
Comment (appflowy) [inline admin_frontend/src/web_api.rs:60]: The `appflowy-flutter://login-callback` requires documentation to explain its usage. One issue to note is that the URL will become invalid if we alter the schema. For now, we can proceed with it.

File: admin_frontend/templates/components/invite.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <tr>
+      <td></td>
+      <td style="text-align: right">
+        <button class="button cyan" id="inviteBtn">Invite</button>
+      </td>
+    </tr>
+  </table>
+
+  <script>
+    document.getElementById("inviteBtn").addEventListener("click", function () {
```
Comment (appflowy) [inline admin_frontend/templates/components/invite.html:23]: If we choose not to use any web frameworks (like React) here, we may need to refactor the HTML

File: admin_frontend/src/web_api.rs (hunk 1 above)
Comment (speed2exe) [reply admin_frontend/src/web_api.rs:60]: I'll add some comments to clarify this part

File: admin_frontend/templates/components/invite.html (hunk 2 above)
Comment (speed2exe) [reply admin_frontend/templates/components/invite.html:23]: Most of the HTML are components and can be reused when necessary to keep the code dry. Is there a specific part you are referring to?

Comment (appflowy) [reply admin_frontend/templates/components/invite.html:23]: No, it's just a concern

**Pull request merged**